test/intl402/Temporal/TimeZone/prototype/getOffsetNanosecondsFor/nanoseconds-subtracted-or-added-at-dst-transition.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2022 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.timezone.prototype.getoffsetnanosecondsfor
+description: >
+  Test offset when nanoseconds are subtracted or added from DST transition.
+features: [Temporal]
+---*/
+
+// From <https://github.com/eggert/tz/blob/main/northamerica>:
+//
+// # Rule  NAME  FROM  TO    -  IN   ON       AT    SAVE  LETTER
+// Rule    CA    1962  1966  -  Oct  lastSun  2:00  0     S
```

Comment:
Small detail, but isn't this the wrong line for the data used in the test below? We need the one about the [last Sunday in April](https://github.com/eggert/tz/blob/main/northamerica#L559)

Reply:
Thanks for catching this error!